app/services/supplementary-billing/reissue-invoices.service.js

```diff
@@ -0,0 +1,68 @@
+'use strict'
+
+/**
+ * Handles the reissuing of invoices
+ * @module ReissueInvoicesService
+ */
+
+const BillingInvoiceModel = require('../../models/water/billing-invoice.model.js')
+const BillingInvoiceLicenceModel = require('../../models/water/billing-invoice-licence.model.js')
+const BillingTransactionModel = require('../../models/water/billing-transaction.model.js')
+const FetchInvoicesToBeReissuedService = require('./fetch-invoices-to-be-reissued.service.js')
+const ReissueInvoiceService = require('./reissue-invoice.service.js')
+
+/**
+ * Handles the reissuing of invoices
+ *
+ * We get all invoices to be reissued for the given billing batch (ie. same region, and marked for reissuing). For each
+ * one of these we call `ReissueInvoiceService` which handles the actual reissuing process and collect the returned
+ * invoice, invoice licence and transaction data which we batch persist one all invoices have been reissued
+ *
+ * @param {module:BillingBatchModel} originalBillingBatch The billing batch to check for invoices to be reissued
+ * @param {module:BillingBatchModel} reissueBillingBatch The billing batch that the reissued invoices will be created on
+ *
+ * @returns {Boolean} `true` if any invoices were reissued; `false` if not
+*/
+
+async function go (originalBillingBatch, reissueBillingBatch) {
```

Comment:
This comment is going to apply to the use of `originalBillingBatch` both here and in `ReissueInvoiceService`.

I don't believe we need it. The 'source' for what to reissue comes from `FetchInvoicesToBeReissuedService` and all it needs is the region ID.

We can get that from `reissueBillingBatch` which then makes `originalBillingBatch` redundant here.